plugins/woocommerce-admin/client/marketplace/components/woo-connect-plugin/woo-connect-plugin.scss

```diff
@@ -46,16 +46,12 @@
 	}
 }
 
-.woocommerce-marketplace__woo-connect-plugin__notice {
-	&:last-child {
-		margin-bottom: $grid-unit-50;
-	}
+.woocommerce-marketplace__woo-connect-plugin__notices {
```

Comment:
Same as for the variable names, how about using woo update manager instead of woo connect in class names?

Reply:
Fixed with https://github.com/woocommerce/woocommerce/pull/44279/commits/121ef048b1ac0caecef6d55ac44fef9c768697ad